src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+public class Task {
+    private boolean isDone;
+    private String taskName;
+    static String done = "[X] ";
```

Comment:
The variables 'done' and 'notDone' are constants, and should be fully capitalized.